apps/studio/components/interfaces/DataWarehouse/WarehouseCollectionDetail.tsx

```diff
@@ -81,23 +124,80 @@ export const WarehouseCollectionDetail = () => {
   const results = formatResults(queryData?.result)
 
   function loadMore() {
-    setPagination({ ...pagination, offset: pagination.offset + pagination.limit })
+    setFilters({ ...filters, offset: filters.offset + filters.limit })
   }
 
   const isLoading = queryLoading || collectionsLoading || isRefetching
 
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>) {
+    e.preventDefault()
+    setFilters({ ...filters, search })
+    refetch()
+  }
+
+  const queryUrl = `/project/${projectRef}/logs/explorer?q=${encodeURIComponent(
+    params.sql || ''
+  )}&source=warehouse`
+
   return (
     <div className="relative flex flex-col flex-grow h-full">
       <ShimmerLine active={isLoading} />
       <LoadingOpacity active={isLoading}>
         <div className="flex flex-col w-full">
-          <div className="flex justify-between items-center pr-3">
-            <h2 className="p-3">{collection?.name}</h2>
+          <div className="flex justify-between items-center h-12 px-5">
+            <form onSubmit={handleSubmit} className="flex items-center gap-2">
+              <h2 className="text-foreground-light">{collection?.name}</h2>
+              <Tooltip>
```

Comment:
use `<ButtonTooltip>` for this - the tooltip wouldn't show up if the button is disabled if used like this

Reply:
IDK why but it's not working for me. Will take a look and fix in a different PR.